tests/functional/disaster-recovery/regional-dr/conftest.py

```
@@ -201,9 +203,9 @@ def factory(replica, compression):
                 resource_name=pool_name,
             )
             if pool_ocp.is_exist(resource_name=pool_name):
-                log.info(
-                    f"Pool {pool_name} already exists on {cluster_name},"
-                    f" skipping creation"
+                log.warning(
+                    f"CephBlockPool {pool_name} already exists on the cluster {cluster_name}. "
+                    f"CephBlockPool configuration is not validated"
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)